cmd/drift.go

```diff
@@ -53,12 +54,20 @@ var (
 			}
 			defer c.Client().Close()
 
-			return c.CallModule(ctx, console.ModuleCallRequest{
+			err = c.CallModule(ctx, console.ModuleCallRequest{
 				Name:          driftModuleID,
 				Params:        driftParams,
 				ModConfigPath: driftConfigPath,
 				OutputPath:    driftOutputPath,
 			})
+			if err != nil {
+				if ee, ok := err.(*console.ExitCodeError); ok {
```

Comment:
This code could be simplified if we decide we only need just one nonzero exit code.